### DISCUSSION TITLE
netvips difference in processing Mac vs Linuc

### DISCUSSION THREAD

**Discussion by mgbee8** [libvips/libvips#4019] [Q&A] [answered]
We are using netvips to tile digital pathology images

We are using the following to tile 

                openslide = Image.NewFromFile(tempFileName);
                if (openslide == null)
                    throw new Exception("openslide is null");
                using Image newImage = openslide.IccTransform("srgb", null, Enums.Intent.Perceptual, null, true);

                newImage.Dzsave(finalPath, uuid.ToString(), tileSize: tileSize);

When we run on Mac the tiles display properly in Chrome, when run on ubuntu linux on kubernetes the prfile is different
Also the tiling is not the same both are using identical tile size of 2048

![linux_version](https://github.com/libvips/libvips/assets/637006/9de3338e-cfe9-441f-a3c1-427c827a872d)
![mac_version](https://github.com/libvips/libvips/assets/637006/49e75364-59a6-4267-9999-01561f88e71f)
I am probably doing something stupid  any help would be appreciated

**Comment by lovell** [comment]
Are you using exactly the same version of libvips built with the same dependencies with the same versions on both platforms?

**Reply by jcupitt** [reply, marked as answer]
Yes, exactly, ICC profile support was only added to openslide in v4.0 at the end of last year, and has not yet reached the official ubuntu repositories.

Unless you have built openslide and libvips from source yourself, you will be using an old version without this feature.

**Reply by mgbee8** [reply]
Thanks for the quick response
